Validation Rules Result Screen - valid (EXPOSUREAPP-8031)

BMItr opened the pull request:
This PR addresses EXPOSUREAPP-8031 (validation Result Screen - valid)
PR will be set to **DRAFT** because Button & Date - Wiring needed after Repo and Screens before this one are completed/merged (see EXPOSUREAPP-7738 for the flow)

Screenshot (scrolled down a little):

![screen](https://user-images.githubusercontent.com/46747780/124009749-447de300-d9de-11eb-884a-d799a13fc4dd.png)

BMItr committed:
ui pattern for rule validation result (valid) - wip

BMItr committed:
clean layout

BMItr committed:
more details

BMItr committed:
figma details adjusted

BMItr committed:
Merge branch 'release/2.6.x' into feature/8031-validation-result-screen-valid

BMItr committed:
merge conflict resolved

BMItr committed:
created module, viewmodel etc

BMItr committed:
fix binding

BMItr committed:
Bind viewmodel

BMItr committed:
improved layout to flow

BMItr committed:
working dark mode, little inner layout adjustment

BMItr committed:
Added clickable links

BMItr committed:
darkmode fix

BMItr committed:
reverted OverviewFramgment (for testing)

BMItr committed:
added Tools context

BMItr committed:
typo

BMItr committed:
Merge branch 'release/2.6.x' into feature/8031-validation-result-screen-valid

BMItr committed:
soruce clean

ChristinaTsigganos commented on Corona-Warn-App/src/main/res/values-de/covid_certificate_strings.xml:
```diff
@@ -102,4 +102,34 @@
+    <!-- XTXT: Validation Rule Result Valid - Header  -->
+    <string name="validation_rules_result_valid_header">Gültiges Zertifikat</string>
+    <!-- XTXT: Validation Rule Result Valid - Result  -->
+    <string name="validation_rules_result_valid_result_title">Ihr Zertifikat ist im gewählten Reiseland gültig.</string>
```
Please change "Reiseland" to "Land", as displayed in Figma: https://www.figma.com/file/KuDEcMSREg3VhZRhC3FXTY/COVID19_Master_2.2?node-id=24596%3A21940

ChristinaTsigganos commented on Corona-Warn-App/src/main/res/values/covid_certificate_strings.xml:
```diff
@@ -101,5 +101,35 @@
+    <!-- XTXT: Validation Rule Result Valid - Header  -->
+    <string name="validation_rules_result_valid_header">Gültiges Zertifikat</string>
+    <!-- XTXT: Validation Rule Result Valid - Result  -->
+    <string name="validation_rules_result_valid_result_title">Ihr Zertifikat ist im gewählten Reiseland gültig.</string>
```
same here, please change "Reiseland" to "Land".

ChristinaTsigganos reviewed (changes requested):
Please adjust "Reiseland" to "Land", apart from that the texts are approved by UA

BMItr committed:
adjusted strings to newest figma

BMItr committed:
Merge branch 'release/2.6.x' into feature/8031-validation-result-screen-valid

vaubaehn commented:
@harambasicluka @ChristinaTsigganos 

I'd suggest a small text enhancement for the 4th bullet point (even the text might also still be draft):

`Ob die im Zertifikat eingetragenen Daten richtig sind, wird `**von der Corona-Warn-App**` nicht geprüft.`

It's because the first three bullet points deal "with things outside the scope" of CWA. Especially the 3rd point is handling with "a verifier app". Hence, at least for me, the last bullet sounds like the verifier app won't check the contents of the certificate. Adding "von der Corona-Warn-App" clears it up imho, and hopefully _all_ users get it right then...

BMItr committed:
Committer: Paddy92 Corrected naming

BMItr committed:
Prepared country date info text

BMItr committed:
Space: the final frontier. These are the voyages of the starship Enterprise...

BMItr committed:
Merge branch 'release/2.6.x' into feature/8031-validation-result-screen-valid

harambasicluka commented:
> @harambasicluka @ChristinaTsigganos
> 
> I'd suggest a small text enhancement for the 4th bullet point (even the text might also still be draft):
> 
> `Ob die im Zertifikat eingetragenen Daten richtig sind, wird `**von der Corona-Warn-App**` nicht geprüft.`
> 
> It's because the first three bullet points deal "with things outside the scope" of CWA. Especially the 3rd point is handling with "a verifier app". Hence, at least for me, the last bullet sounds like the verifier app won't check the contents of the certificate. Adding "von der Corona-Warn-App" clears it up imho, and hopefully _all_ users get it right then...
> 
> Edit: Or move the 4th bullet to the beginning as first bullet, and write as
> `Ob die im Zertifikat eingetragenen Daten richtig sind, wird `**hier**` nicht geprüft.` Then the coherence of the following three bullets fits also well.

Thanks @vaubaehn we are looking into this 👍

ChristinaTsigganos commented:
> @harambasicluka @ChristinaTsigganos
> 
> I'd suggest a small text enhancement for the 4th bullet point (even the text might also still be draft):
> 
> `Ob die im Zertifikat eingetragenen Daten richtig sind, wird `**von der Corona-Warn-App**` nicht geprüft.`
> 
> It's because the first three bullet points deal "with things outside the scope" of CWA. Especially the 3rd point is handling with "a verifier app". Hence, at least for me, the last bullet sounds like the verifier app won't check the contents of the certificate. Adding "von der Corona-Warn-App" clears it up imho, and hopefully _all_ users get it right then...
> 
> Edit: Or move the 4th bullet to the beginning as first bullet, and write as
> `Ob die im Zertifikat eingetragenen Daten richtig sind, wird `**hier**` nicht geprüft.` Then the coherence of the following three bullets fits also well.

Thanks a lot, @vaubaehn. We checked and decided to consider your first suggestion to add the information  "von der Corona-Warn-App" to the last bullet point. Thanks a lot for your valuable contribution to the CWA.

ChristinaTsigganos committed:
Add text for clarity (#3664)

Adding "in der Corona-Warn-App" to the sentence for more clarity after we received feedback from the community here.

BMItr commented:
This PR will probably be closed and replaces by another due new Common-Model (new Base Branch).
Will clean it when its done.
No more review required here.

BMItr commented:
Follow UP: https://github.com/corona-warn-app/cwa-app-android/pull/3678
